greenplumpython/op.py

```diff
@@ -0,0 +1,91 @@
+"""
+Indices are essential for fast data searching. With indices, we can retrieve
+rows we want by scanning the index, rather than the entire dataframe. As a
+result, when the dataframe is large, the amount of data to be scanned is
+typically mich smaller with index.
+
+In pandas, a :code:`DataFrame` or :code:`Series` can have only one index. This
+means that we can do index scan based on one column. While for other columns,
+we will have to go through all rows. This can be rather inefficient.
+
+Backed by database systems, GreenplumPython overcomes this limitation by
+allowing creating
+
+- multiple indices, each for one set of columns, and
```

Comment:
On VMware Greenplum Documentation, the plural formal of index is **indexes**, maybe we should standardise with it.